common/lib/capa/capa/responsetypes.py

```diff
@@ -250,8 +251,15 @@ def render_html(self, renderer, response_msg=''):
           - renderer : procedure which produces HTML given an ElementTree
           - response_msg: a message displayed at the end of the Response
         """
-        # render ourself as a <span> + our content
-        tree = etree.Element('span')
+
+        # get responsetype index to make responsetype label
+        response_index = self.xml.attrib['id'].split('_')[-1]
+
+        # wrap the content inside a section
+        tree = etree.Element('section')
+        tree.set('class', 'wrapper-problem-response')
+        tree.set('tabindex', '-1')
+        tree.set('aria-label', RESPONSE_LABEL + ' ' + response_index)
```

Comment:
Ah, I see. I was thinking of the value being an ID, which is why I was confused about how a space could be in it.

Are we trying to translate this text? If so, appending the index will not translate well at all, and a placeholder should be used instead.